cmd/rsh.go

```diff
@@ -141,9 +151,26 @@ func (r *iexec) containerPrompt() error {
 		return nil
 	}
 
+	templates := &promptui.SelectTemplates{
+		Active:   fmt.Sprintf("Container: %s {{ . | cyan }}", promptui.IconSelect),
+		Inactive: "Container: {{ . | magenta }}",
+		Selected: fmt.Sprintf("Container: %s {{ . | cyan }}", promptui.IconGood),
+	}
+
+	log.Debugf("naked: %v", naked)
+	if naked {
```

Comment:
how come you have this same if statement repeated as on line 118?

Reply:
We can look at this and discuss what solutions would look like. The `if` conditional is the same but the templates are different.